pkg/reconciler/v1alpha1/eventlistener/eventlistener_test.go

```diff
@@ -285,32 +400,40 @@ func logConfig(ns string) *corev1.ConfigMap {
 	return lc
 }
 
-var withStatus = bldr.EventListenerStatus(
-	bldr.EventListenerConfig(generatedResourceName),
-	bldr.EventListenerAddress(listenerHostname(generatedResourceName, namespace, *ElPort)),
-	bldr.EventListenerCondition(
-		v1alpha1.ServiceExists,
-		corev1.ConditionTrue,
-		"Service exists", "",
-	),
-	bldr.EventListenerCondition(
-		v1alpha1.DeploymentExists,
-		corev1.ConditionTrue,
-		"Deployment exists", "",
-	),
-	bldr.EventListenerCondition(
-		apis.ConditionType(appsv1.DeploymentAvailable),
-		corev1.ConditionTrue,
-		"Deployment has minimum availability",
-		"MinimumReplicasAvailable",
-	),
-	bldr.EventListenerCondition(
-		apis.ConditionType(appsv1.DeploymentProgressing),
-		corev1.ConditionTrue,
-		fmt.Sprintf("ReplicaSet \"%s\" has successfully progressed.", eventListenerName),
-		"NewReplicaSetAvailable",
-	),
-)
+var withStatus = func(port ...int) bldr.EventListenerOp {
+	var p int
+	if len(port) == 0 {
+		p = *ElPort
+	} else {
+		p = port[0]
+	}
+	return bldr.EventListenerStatus(
+		bldr.EventListenerConfig(generatedResourceName),
+		bldr.EventListenerAddress(listenerHostname(generatedResourceName, namespace, p)),
```

Comment:
very minor: instead of modifying `withStatus` to take in an parameter, we can create a `withTLSPort` function that sets the EventListenerAddress to the right port. We'd call it using `makeEL(withStatus, withTLSPort, func...)`

Reply:
Modified :+1: